GNUmakefile

```diff
@@ -1,23 +1,30 @@
 default: testacc
 
 plugin_version = 99.99.99
-plugins_mirror_path = ~/Library/Application\ Support/io.terraform/plugins
-plugins_provider_path = registry.redislabs.com/redislabs/rediscloud/$(plugin_version)/darwin_amd64/
+plugins_mirror_path = ~/.terraform.d/plugins
+plugins_provider_path = registry.terraform.io/RedisLabs/rediscloud/$(plugin_version)/darwin_amd64/
```

Comment:
`darwin_amd64` should probably be a variable set depending on the running environment

Reply:
Ah missed this and adjusted to make each part of the path a variable.  GOOS and GOARCH are used to populate.